core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -278,6 +278,13 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val IGNORE_PROCFS_TREE_METRICS_GETTER_WARNING =
```

Comment:
Let's not make a flag for this. Reduce the log level

Reply:
Done